scripts/compile-pytorch-ipex.sh

```diff
@@ -216,24 +224,24 @@ build_ipex() {
   if [ ! -d "$IPEX_PROJ" ]; then
     echo "**** Cloning $IPEX_PROJ ****"
     cd $BASE
-    if [ "$FAKE_IPEX" = true ]; then
+    if [ "$NO_OP_IPEX" = true ]; then
       mkdir intel-extension-for-pytorch
       cd intel-extension-for-pytorch
       cat > setup.py <<EOF
 from setuptools import setup
 
 name = "intel-extension-for-pytorch"
-version = "0.1+FAKE"
+version = "2.4.0+dummy_no-op"
```

Comment:
Per https://peps.python.org/pep-0440/#local-version-identifiers Python local version identifier can contain only:
* ASCII letters ([a-zA-Z])
* ASCII digits ([0-9])
* periods (.)

I would recommend replacing 
```suggestion
version = "2.4.0+noop"
```